keras_fsl/metrics/gram_matrix_metrics.py

```diff
@@ -7,7 +7,8 @@
 
 def top_score_classification_accuracy(y_true, y_pred):
     """
-    Use the top score of an image against all the samples from the same class to get a score per class for each image.
+    Use the top score of a sample against all the samples from the same class to get a score per class for each image.
```

Comment:
Use the top score of a sample against all the other sample to get a predicted label for each sample. Then mean accuracy is returned.

Note: if there is no other sample of the same class, the sample will always be counted as failure since it is not possible to find the right class in the other samples.